src/css/tabs/osd.css

```diff
@@ -517,16 +527,31 @@ button {
   }
 }
 
-.tab-osd .grid-col.col4:first-child, .tab-osd  .grid-col.col4:last-child {
-    width: calc((100% - 390px) / 2);
+.tab-osd .grid-row {
+    justify-content: center;
+    gap: 7px;
+}
+
+.tab-osd .grid-row .elements {
+  flex: 1 1 auto;
+  width: 210px;
+  max-width: 700px;
 }
 
-.tab-osd .grid-row {
-    margin-bottom: 40px;
+.tab-osd .grid-row .osd-preview {
+  flex: 2 0 auto;
+  width: 350px;
+  max-width: max(min(950px,calc((100vh - 470px) * 1.25)),350px);
+}
+
+.tab-osd .grid-row .osd-profile {
+  flex: 1 1 auto;
+  width: 210px;
```

Comment:
On Windows/Linux: 180px is enough.

Reply:
You are thinking like an "English" user. There are tons of languages, so this:
- Is a reasonable width for any language. Some of them will be cut, but we can't do it better.
- Is a size that fits in the minimum width of the window without breaking the layout.